### PR TITLE
ON-32691 # Added Lookups class

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: src/classes/Lookups.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+   * })
+   * // Use lookups here
+   * ```
+   *
+   * @param options
+   * @returns
+   */
+  searchLookups(options: {
+    /** The exact id of the lookup you wish to get */
+    organisationId: string
```
Comment (mymattcarroll) [inline src/classes/Lookups.ts:62]: No need to pass this, the key is scoped to the organisation.

File: src/classes/Lookups.ts [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+   * ```typescript
+   * const newLookup = {
+   *   id: 1,
+   *   createdAt: '2023-06-28T02:00:03.000Z',
+   *   updatedAt: '2023-06-28T02:00:03.000Z',
+   *   name: 'My New Lookup',
+   *   environments: [
+   *     {
+   *       formsAppEnvironmentId: 1,
+   *       urL: 'https://my-url.com/lookup',
```
Comment (mymattcarroll) [inline src/classes/Lookups.ts:128]:
```suggestion
   *       url: 'https://my-url.com/lookup',
```

File: src/classes/Lookups.ts [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  /**
+   * #### Example
+   *
+   * ```typescript
+   * const newLookup = {
+   *   name: 'My New Lookup',
+   *   environments: [
+   *     {
+   *       formsAppEnvironmentId: 1,
+   *       urL: 'https://my-url.com/lookup',
```
Comment (mymattcarroll) [inline src/classes/Lookups.ts:92]:
```suggestion
   *       url: 'https://my-url.com/lookup',
```

File: src/classes/Lookups.ts (hunk 1 above)
Comment (Zaxist) [reply src/classes/Lookups.ts:62]: Endpoint requires it

Comment (mymattcarroll) [reply src/classes/Lookups.ts:62]: we should update the endpoint.

Comment (Zaxist) [reply src/classes/Lookups.ts:62]: We discussed yesterday and decided not to for now. We can discuss it again if you'd like?

**Pull request merged**